src/api/arithmetic.md

```diff
@@ -1 +1,106 @@
 # Arithmetic
+
+Let's suppose you are creating an finance app for people who are above 18 years old. Normally, instead of asking the age itself, you would ask for the birthdate to track the age evolution, rigth?
```

Comment:
Age may not be the best example, as all years are not the same length. Perhaps something occurring every so many days of weeks?

Reply:
That's a good point. What do you think about calculating the next payment date of a monthly subscription?

We could add an example of adding 30 days in a date, considering different date lengths. We could also consider when the current month is December and the result would be a date in the next year to show how to handle these "corner" cases.

I'm thinking of using Duration to add the amount of days in a Date object with the .add_days()

What are your thoughts on that?

Reply:
Or maybe calculating the date of arrival of an online purchase 🤔

Reply:
Arrival for a purchase would work! That's something that wouldn't care about the lengths of months/years.

Reply:
Great! I'll come up with an example and we can work upon that :)

Reply:
Just commited the change in the example :)